src/RolloutScorer/RolloutScorerAzureFunction/RolloutScorerAzureFunction.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="Current">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AzureFunctionsVersion>v2</AzureFunctionsVersion>
```

Comment:
I really don't like an Azure function for this. :-(  It's yet a third kind of thing we need to deploy in arcade-services... it's becoming a bit of a grab-bag of tech at this point.

I think we either need to decide on ServiceFabric or AzureFunctions, and stick with it.  Having both is hurting our maintence.  Could we make this an... Actor or whatever they are supposed to be and stick it in the... TelemetryApplication? This is... basically telemetry.  It's release times and issues seem conceptually related to build times and failures that I wouldn't blink if they were combined.

Reply:
Using an Azure function was the design proposed in the design doc. However, if it's trivial to switch this to a Service Fabric deal i'm happy to do that. Do you have any pointers for converting?

Reply:
Unfortunately I don't think it's trivial.  It's probably a bigger discussion with the team around what tech's we want to centralize on. I'd leave it like this for now.

Reply:
Okay that sounds fine to me. I'll resolve this for now; I'm definitely interested in being involved in that conversation down the line.